modules/mapbox/src/deck-utils.ts

```diff
@@ -115,26 +117,25 @@ export function removeDeckInstance(map: Map & {__deck?: Deck | null}) {
   map.__deck = null;
 }
 
-export function getInterleavedProps(currProps: DeckProps) {
-  const nextProps: DeckProps = {
-    ...currProps,
-    // TODO: remove 'any' cast
-    parameters: {
-      depthMask: true,
-      depthWriteEnabled: true,
-      depthCompare: 'less-equal',
-      blend: true,
-      blendFunc: [GL.SRC_ALPHA, GL.ONE_MINUS_SRC_ALPHA, GL.ONE, GL.ONE_MINUS_SRC_ALPHA],
-      polygonOffsetFill: true,
-      depthFunc: GL.LEQUAL,
-      blendEquation: GL.FUNC_ADD,
-      ...currProps.parameters
-    } as any,
-    // @ts-ignore views prop is hidden by the types because it is not expected to work the same way as in standalone Deck, see documentation
-    views: currProps.views || [new MapView({id: 'mapbox'})]
-  };
-
-  return nextProps;
+export function getDefaultParameters(map: Map, interleaved: boolean): Parameters {
+  const result: Parameters = interleaved
+    ? {
+        depthWriteEnabled: true,
+        depthCompare: 'less-equal',
+        blend: true,
+        blendColorSrcFactor: 'src-alpha',
+        blendColorDstFactor: 'one-minus-src-alpha',
+        blendAlphaSrcFactor: 'one',
+        blendAlphaDstFactor: 'one-minus-src-alpha',
+        blendColorOperation: 'add',
+        blendAlphaOperation: 'add',
+        polygonOffsetLine: true
```

Comment:
`depthMask` and a number of other parameters have been removed, is that intentional?

Reply:
Yes. These are the ones removed in order to satisfy luma core `Parameters` typing:
- `depthMask` (equivalent to `depthWriteEnabled`)
- `depthFunc` (replaced by `depthCompare`)
- `polygonOffsetFill` (replaced by `depthBias`)
- `blendFunc` (replaced by `blend*Factor`)
- `blendEquation` (replaced by `blend*Operation`)